docs/learn/export-pipeline/data-value-extractor.md

```diff
@@ -11,6 +11,17 @@ import WIP from '../\_wip.md';
 
 ## Installation
 
+1. Login to your Rossum account.
+1. Navigate to **Extensions → My extensions**.
+1. Click on **Create extension**.
+1. Fill the following fields:
+   1. Name: `Data value extractor`
+   1. Trigger events: `Export`
+   1. Extension type: `Webhook`
+   1. URL (see below)
+1. In "Advanced settings" select **Token owner** (should have Admin access)
+1. Click **Create the webhook**.
+
 <WIP />
```

Comment:
@VaclavRut Do you think it would make sense to remove the `WIP` tag already since the installation instructions are here now?